src/MyEventEmitter.js

```diff
@@ -1,12 +1,71 @@
 'use strict';
 
-export class MyEventEmitter {
-  on() {}
-  once() {}
-  off() {}
-  emit() {}
-  prependListener() {}
-  prependOnceListener() {}
-  removeAllListeners() {}
-  listenerCount() {}
+class MyEventEmitter {
+  constructor() {
+    this.listeners = [];
+  }
+
+  on(listenerName, callback) {
+    this.listeners.push({
+      listenerName,
+      callback,
+      once: false,
+    });
+  }
+
+  once(listenerName, callback) {
+    this.listeners.push({
+      listenerName,
+      callback,
+      once: true,
+    });
```

Comment:
Instead of adding `once` property you can create a wrapper callback that will execute callback and then off it and add it instead.
```javascript
const onceWrapper = () => {
  callback();
  this.off(eventName, onceWrapper);
};
```